### ISSUE TITLE
f5 overrides default sort bind on Sublime Text

### ISSUE THREAD

**Issue by wbotelhos** [FichteFoll/InsertDate#21]
Could you change the insert command key bind?:

```json
{ "keys": ["f5"], "command": "insert_date_panel" }
```

`f5` is the default bind to sort lines.

**Comment by FichteFoll** [comment]
This is true, but only for OSX. Do you have a suggestion for a different binding?

**Comment by wbotelhos** [comment]
Thanks for your answer, @FichteFoll.

I used to set `ctrl + 1`, `ctrl + 2` and so...

**Comment by FichteFoll** [comment]
ctrl+1 ... are also bound by default, on both OSX and Linux/Windows, so that's not an option.

Would you happen to know if there is a more or less common hotkey for inserting the current date in other applications on OSX? I think I grabbed the F5 binding from TextPad, but don't actually know if this is common for other applications since F5 is mostly "refresh".

**Comment by wbotelhos** [comment]
May add the `shift` key. I tried here and there is no OSX and Linux bind for `ctrl + shift + 1..`

**Comment by FichteFoll** [comment]
I would rather use the shift key with F5 instead to be more consistent, ctrl+shift+1 seems very arbitrary.

**Comment by Snake-Sanders** [comment]
how do you change the default key (on Win), do you need to create a new conf user file or you directly uses insert_date.sublime-settings ? I tried the last one but the syntax do not work. I also tried to wrap it within [,]

```
{
    "tz_in": "Europe/Vienna",
    "keys": ["f6"], "command": "insert_date_panel"
}
```
